backend/services/AuthService.js

```diff
@@ -0,0 +1,59 @@
+const axios = require('axios');
```

Comment:
axios may not be allowed you know, but i think we can leave it for now

Reply:
this is the backend they don't really care